app/models/course/forum/topic.rb

```diff
@@ -79,11 +77,14 @@ class Course::Forum::Topic < ApplicationRecord
           group('course_discussion_posts.topic_id').
           where(topic_id: topic_ids)
 
-    last_posts = Course::Discussion::Post.with_creator.where('id in (?) or id in (?)', min_ids, max_ids)
+    last_posts = Course::Discussion::Post.with_creator.
+          where('course_discussion_posts.id in (?) or course_discussion_posts.id in (?)', min_ids, max_ids)
```

Comment:
Why the change in expression? Does the previous one not work in Rails 7?

Reply:
this is weird. I was working on this while doing the change in Rails 7, and the error was thrown here mentioning that `id is ambiguous` (between the `id` of `course_discussion_posts` and the `id` of `course_forum_topics`), but seems like after I try again, this error is gone already.

Reverting back to the original one.